www/install/php/Update-20.04.1.php

```diff
@@ -45,6 +45,11 @@
     );
     $engineCommand = $res->fetch()['command_file'];
 
+    // escape double quotes and backslashes
+    $needle = ['\\', '"'];
+    $escape = ['\\\\', '\"'];
+    $password = str_replace($needle, $escape, password);
```

Comment:
$password = str_replace($needle, $escape, **password**);
Where does the word "password" come from???

Reply:
Ok it's the password constant

Reply:
yes it is... sorry I should have commented it